### PR TITLE
docs: add a `cmd.exe` version of the `cd` snippet

### PR DESCRIPTION
**Pull request by mataha**
Hello,

I would like to include a snippet that presents usage of git combined with `onefetch` in `cmd.exe`, as I felt Windows shells were heavily underrepresented on that front.

It is suited for use in `AutoRun` scripts (something akin to `.bashrc` or `.bash_profile`, but isn't enabled by default).

### CODE REVIEW

File: .github/wiki/Getting-Started.md [hunk 1]
```diff
@@ -66,6 +66,15 @@ funcsave cd
 funcsave check_directory_for_new_repository
 ```
 
+By [**@mataha**](https://github.com/mataha)
```
Comment (spenserblack) [inline .github/wiki/Getting-Started.md:69]: Minor (non-blocking) nitpick: we probably want all of these snippet credits to have the same style, so we don't show favor for one author over another. So maybe either update this line's style, or update the style for all of the other author credits.

**Review by spenserblack — APPROVED** [review]
This is far beyond my knowledge of Windows cmd.exe, but LGTM :laughing:

**Review by o2sh — APPROVED** [review]
Hey @mataha, awesome script! Too bad that I can't add it to my $PROFILE as a PowerShell script, but thank you so much for sharing it 🙏. Your contribution is greatly appreciated! 👍

I'll let you address @spenserblack remark before merging it.

File: .github/wiki/Getting-Started.md [hunk 2]
```diff
@@ -66,6 +66,15 @@ funcsave cd
...
+
+An adaptation of the above snippet suited for Windows's `cmd.exe`,
+specifically for inclusion in `AutoRun` or DOSKEY scripts:
+```cmd
+@set LAST_REPOSITORY=
+@doskey cd = @(if "$*"=="" (chdir /d "%%HOMEDRIVE%%%%HOMEPATH%%") else (chdir /d $*)) ^&^& (for /f "usebackq tokens=* delims=" %%i in (`"git rev-parse --show-toplevel 2$Gnul"`) do @(if not "%%~i"=="" if not "%%~i"=="%%LAST_REPOSITORY%%" (onefetch ^& set "LAST_REPOSITORY=%%~i")))
```
Comment (o2sh) [inline .github/wiki/Getting-Started.md:75]:
```suggestion
@set LAST_REPOSITORY=

@doskey cd = @(
  if "$*"=="" (
    chdir /d "%%HOMEDRIVE%%%%HOMEPATH%%"
  ) else (
    chdir /d $*
  )
) ^&^& (
  for /f "usebackq tokens=* delims=" %%i in (
    `"git rev-parse --show-toplevel 2$Gnul"`
  ) do @(
    if not "%%~i"=="" if not "%%~i"=="%%LAST_REPOSITORY%%" (
      onefetch ^& set "LAST_REPOSITORY=%%~i"
    )
  )
)
```

Just formatted it to make it more readable

File: .github/wiki/Getting-Started.md (hunk 1 above)
Comment (mataha) [reply .github/wiki/Getting-Started.md:69]: Done.

**Comment by mataha** [comment]
Re: your comment - this can be formatted, but every linebreak has to be escaped **and** the escape has to be in a context in which whitespace characters are not significant, otherwise indentation will cause an error:

```cmd
@set LAST_REPOSITORY=

@doskey cd = @( ^
    if "$*"=="" ( ^
        chdir /d "%%USERPROFILE%%" ^
    ) else ( ^
        chdir /d $* ^
    ) ^
) ^&^& ( ^
    for /f "usebackq tokens=* delims=" %%i in ( ^
        `"@git rev-parse --show-toplevel 2$Gnul"` ^
    ) do @( ^
        if not "%%~i"=="" if not "%%~i"=="%%LAST_REPOSITORY%%" ( ^
            onefetch ^& set "LAST_REPOSITORY=%%~i" ^
        ) ^
    ) ^
)
```

Does it look better? Honestly, I don't know...

**Comment by o2sh** [comment]
>otherwise indentation will cause an error

Yeah that's a shame. I think we can keep it as is 👍 

Thanks for the effort though @mataha

**Pull request merged**